### DISCUSSION TITLE
Could not find method jsonschema2pojo() for arguments

### DISCUSSION THREAD

**Discussion by aftab-mannan** [jsonschema2dataclass/js2d-gradle#162] [Q&A] [answered]
Hi I am getting error post following the step in ReadMe.md. Gradle version 6.9

**Comment by eirnym** [comment]
@aftab-mannan Could you supply a configuration example to reproduce this issue?

**Comment by cbossi** [comment, marked as answer]
I think it's a typo in the documentation: In the `README.adoc` it is `jsonschema2pojo` instead of `jsonSchema2Pojo`. (The `P` of `Pojo` should be uppercase)

**Reply by eirnym** [reply]
It is, PR is welcome

**Reply by cbossi** [reply]
https://github.com/jsonschema2dataclass/js2d-gradle/pull/183
